src/misc/ssEconomy.lua

```diff
@@ -218,9 +228,24 @@ function ssEconomy:calculateLoanInterestRate()
     g_currentMission.missionStats.loanAnnualInterestRate = seasonsYearInterest
 end
 
+function ssEconomy:aiLoad(savegame)
+
+	-- After loading the aiVehicle, convert the vanilla pricePerMS to Seasons values (2000 to 1650 per hour), and store in a new variable, so pricePerMS can be altered without losing the original value.
+	self.originalPricePerMS = self.pricePerMS * g_seasons.economy.aiVanillaToSeasonsMultiplier;
+end
+
 function ssEconomy.aiUpdateTick(self, superFunc, dt)
-    if self:getIsActive() and self.pricePerMS ~= 0 then
-        self.pricePerMS = ssUtil.isWorkHours() and g_seasons.economy.aiPricePerMSWork or g_seasons.economy.aiPricePerMSOverwork
+    if self:getIsActive() then
```

Comment:
If you do what i suggested above, here you can do: `if overwork, factor = aiPriceOverwork/2000`
`else factor = aiPrice/2000`
`self.pricePerMS = self.originalPricePerMS*factor`
This last line needs a check for original price being nil.